src/middlewared/middlewared/alembic/versions/21.MM/2021-08-20_17-20_serial_ports.py

```diff
@@ -0,0 +1,37 @@
+"""
+User serial port(s) name instead of I/O address
+
+Revision ID: 725b7264abe6
+Revises: 29abd3dce632
+Create Date: 2021-20-08 17:20:42.818433+00:00
+"""
+from alembic import op
+import sqlalchemy as sa
+
+from middlewared.utils import osc
```

Comment:
Please do not import anything from middleware in migrations and just copy the code here.